contracts/token/ERC721/extensions/IERC721Enumerable.sol

```diff
@@ -3,18 +3,13 @@
 
 pragma solidity ^0.8.0;
 
-import "../IERC721.sol";
+import "./IERC721Supply.sol";
 
 /**
  * @title ERC-721 Non-Fungible Token Standard, optional enumeration extension
  * @dev See https://eips.ethereum.org/EIPS/eip-721
  */
-interface IERC721Enumerable is IERC721 {
-    /**
-     * @dev Returns the total amount of tokens stored by the contract.
-     */
-    function totalSupply() external view returns (uint256);
-
+interface IERC721Enumerable is IERC721Supply {
```

Comment:
This is going to change the definition of `IERC721Enumerable` and thus the value returned by `type(IERC721Enumerable).interfaceId`. 

So this change is breaking the ERC165 support that is implemented in ERC721Enumerable.
```solidity
    function supportsInterface(bytes4 interfaceId) public view virtual override(IERC165, ERC721) returns (bool) {
        return interfaceId == type(IERC721Enumerable).interfaceId || super.supportsInterface(interfaceId);
    }
```